readme.md

```diff
@@ -1,19 +1,61 @@
 <img align="right" src="logo-256.png"/>
 
-Common Package Specification
-============================
-
-This is the repository for the Common Package Specification; a mechanism for
-describing a software package such that it may be easily consumed by other
-software projects.
-
-The actual specification may be viewed [here](https://cps-org.github.io/cps/),
-or by viewing the latest
-[source documents](https://github.com/cps-org/cps/).
-An example CPS file can be seen
-[here](https://github.com/cps-org/cps/blob/master/sample.cps).
-
-If you have [Sphinx](http://www.sphinx-doc.org/) installed, the provided
-`Makefile` may also be used to render the latest version of the specification
-locally. (Or run `sphinx-build` directly with your favorite options. Note
-however that non-HTML output is not tested and may not work.)
+# Common Package Specification (CPS)
+
+The **Common Package Specification (CPS)** is an open specification for describing software packages and their dependencies in a language-agnostic, tool-agnostic, and platform-agnostic way. CPS aims to provide a unified, extensible, and machine-readable format for package metadata, making it easier to share, build, and consume libraries across diverse build systems and environments.
+
+## Key Features
+
+- **Standardized Format:** Simple, declarative YAML or JSON for package metadata, dependencies, build flags, and more.
```

Comment:
I think we shoulod 
```suggestion
- **Standardized Format:** Simple, declarative JSON for package metadata, dependencies, build flags, and more.
```

@mwoehlke Correct me if I'm being inaccurate here, but in my mind everyone's required to use standard JSON to ensure interoperability.